pkgs/top-level/python-packages.nix

```
@@ -13513,6 +13517,8 @@ self: super: with self; {
 
   qmk-dotty-dict = callPackage ../development/python-modules/qmk-dotty-dict { };
 
+  qwt = callPackage ../development/python-modules/qwt { };
```

```suggestion
  pythonqwt = callPackage ../development/python-modules/pythonqwt { };
```
https://pypi.org/project/PythonQwt/ exists but https://pypi.org/project/qwt/ doesn't

---

> [pypi.org/project/PythonQwt](https://pypi.org/project/PythonQwt/) exists but [pypi.org/project/qwt](https://pypi.org/project/qwt/) doesn't

OK. IIRC in the past we said that `py` and `python-` alike prefixes should be removed from the pname, but the current `python.section.md` docs don't say that anymore.